### ISSUE TITLE
Tests seem to be thoroughly broken on Linux

### ISSUE THREAD

**Issue by DaveCTurner** [kallisti-dev/hs-webdriver#79]
Filenames are lower case, and contents of examples/readme-example-beginner.lhs is just:

    .\readme-example-beginner.md

**Comment by kallisti-dev** [comment]
Yep they currently don't work, but I haven't had time to fix them. I was attempting to use a  script mentioned [here](http://stackoverflow.com/questions/5917249/git-symlinks-in-windows#16754068) to convert my windows symlinks to linux ones, but it must not be working correctly.

**Comment by kallisti-dev** [comment]
The biggest hurdle is ensuring that the test suites are executed in an environment that has the necessary components (Selenium, web browsers, window managers, etc) installed. I may consider using SauceLabs for this purpose. I believe you've worked with SauceLabs before, correct? Would you recommend it for setting up library tests?

**Comment by DaveCTurner** [comment]
Indeed, running this kind of full-stack test is quite a challenge. I wonder if it'd be easier in the long run to have a fake webdriver that most tests can run against, and then have some 'example' scripts that you can run in a real environment that aren't really tests so much as demos. The advantage of the fake webdriver is that you can actually assert that hs-webdriver is working as desired rather than the whole stack (including a remote webserver) is all behaving properly.

I've not worked with SauceLabs before - we run all our Selenium tests in-house.

**Comment by kallisti-dev** [comment]
Creating an entire dummy Selenium server sounds just as error-prone as writing a client. One typo in my HTTP endpoints and suddenly I have tests failing due to the dummy server. But I suppose it's the only way to verify that an issue isn't related to the Selenium project and is an actual problem with the Haskell code.

**Comment by DaveCTurner** [comment]
I'm afraid so. Fortunately the mistakes one tends to make on either side
(typos or otherwise) can be quite orthogonal. I also wonder whether you
could use the online docs to generate at least some of the test code;
particularly the typo-prone bits like endpoint URLs?

**Comment by thomasjm** [comment]
I'm going through and closing old issues. The tests are working fine now.
